@utahdts/utah-design-system/react/components/forms/PlainText.jsx

```diff
@@ -26,6 +28,8 @@ export function PlainText({
   wrapperClassName,
   ...rest
 }) {
+  const internalId = useId();
+  const { value: currentValue } = useFormContextInput({ id: id || internalId, value });
   return (
     <div className={joinClassNames('plain-text-wrapper', 'plain-text-wrapper--plain-text', wrapperClassName)} ref={innerRef}>
```

Comment:
These class wrappers don't align with the input-wrapper classes.
Later on when input-wrapper has styling applied it will mean these are one-off problems to solve.

Reply:
ie `input-wrapper input-wrapper--date-input`
replace with `input-wrapper input-wrapper--plain-text`